build.gradle.kts

```diff
@@ -58,7 +58,9 @@ kotlinDslPluginOptions {
 }
 
 dependencies {
-    implementation("com.amazonaws:aws-java-sdk-s3:1.11.751")
+    implementation(platform("io.netty:netty-bom:4.1.63.Final"))
```

Comment:
```suggestion
```

Reply:
Don't think this is needed for sync clients

Reply:
In practice, v2sdk pulls netty anyway:

```
\--- software.amazon.awssdk:s3 -> 2.16.52
     \--- software.amazon.awssdk:netty-nio-client:2.16.52
          +--- software.amazon.awssdk:annotations:2.16.52
          +--- software.amazon.awssdk:http-client-spi:2.16.52 (*)
          +--- software.amazon.awssdk:utils:2.16.52 (*)
          +--- software.amazon.awssdk:metrics-spi:2.16.52 (*)
          +--- io.netty:netty-codec-http:4.1.63.Final
          |    +--- io.netty:netty-common:4.1.63.Final
          |    +--- io.netty:netty-buffer:4.1.63.Final
...
```

I wonder what happens if I exclude it :)

Reply:
It looks like the tests pass